packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts

```diff
@@ -14,15 +14,15 @@ beforeAll(async () => {
     relationshipId = (await establishRelationship(services1.transport, services2.transport)).id;
 }, 30000);
 
+afterAll(async () => {
+    return await serviceProvider.stop();
```

Comment:
Why did you move this? We always do afterAll after beforeAll, I don't see a reason to do it differently here.

Reply:
Because before it was like

beforeAll
beforeEach
afterAll
afterEach

now it is 

beforeAll
afterAll
beforeEach
afterEach

we also have that in other tests.